Deploy-CAM.ps1

```diff
@@ -64,9 +64,24 @@ param(
 
     [switch]$ignorePrompts,
 
+    [parameter(Mandatory = $false)]
+    $enableRadiusMfa=$null,
```

Comment:
I think it's $null if not specified (like the ones below) but no harm explicitly setting to $null I guess.

Reply:
But if you don't set to null then you can make it a [bool] type which would be cleaner.

Reply:
This was my hacky way of getting three states for this:
1. If True, skip prompts and enable Radius settings
2. If False, skip prompt and disable Radius settings
3. If Null, go through prompts and set as desired by user
Bool limits me to three states, but I can change it and force the prompt unless ignorePrompts is set if you want

Reply:
I can probably change it now that I think about it some more...

Reply:
Yeah I think setting to bool will still allow the three state bit you want (which I agree with) but just get the whole 'did I type the text correctly' part out of the equation.

Reply:
FWIW, I did print out the type of enableRadiusMfa with a call to getType() and it did print bool but that was after it was set to $true/$false.